### DISCUSSION TITLE
Use non-default Appearance for specific page?

### DISCUSSION THREAD

**Discussion by brianlg** [jpanther/congo#177] [Q&A] [answered]
My site is set to "dark" as the default appearance. I am wanting to set my resume.md page to "light" while the rest of the site remains dark. Is this doable, and if so, how would I go about doing this?

**Comment by jpanther** [comment, marked as answer]
Although the theme doesn't directly support overriding the appearance on a per-article basis, you could possibly add some JavaScript to force this behaviour on page load.

Try including something like the below in your `resume.md` content:

```html
<script type="text/javascript">
  document.documentElement.classList.remove("dark");
</script>
```

The caveat with this is that the JS will only execute once the page has loaded so it may appear dark until the code has a chance to run.

**Reply by brianlg** [reply]
Very cool -- that script did the trick, and that works for me. Thank you
